fluss-server/src/main/java/com/alibaba/fluss/server/coordinator/event/CoordinatorEventManager.java

```diff
@@ -53,6 +53,7 @@ public final class CoordinatorEventManager implements EventManager {
 
     // metrics
     private Histogram eventProcessTime;
+    private Histogram eventQueueTime;
 
     private static final int WINDOW_SIZE = 1024;
```

Comment:
I'd like to change `WINDOW_SIZE` to 100 since the coordinator event don't happens very frequently.
```
// since coordinator event don't happens very frequently,
    // set to a smaller window size
    private static final int WINDOW_SIZE = 100;
```